release-notes/v5.5.4.md

```diff
@@ -0,0 +1,16 @@
+
+ #### <sub><sup><a name="3600" href="#3600">:link:</a></sup></sub> feature
+  
+ * Concourse now garbage-collects worker containers and volumes that are not tracked in the database. In some niche cases, it is possible for containers and/or volumes to be created on the worker, but the database (via the web) assumes their creation had failed. If this occurs, these untracked containers can pile up on the worker and use resources. #3600 ensures that they get cleaned appropriately.ed worker heartbeat log level from `debug` to `info` to reduce extraneous log output for operators #4606	* Concourse now garbage-collects worker containers and volumes that are not tracked in the database. In some niche cases, it is possible for containers and/or volumes to be created on the worker, but the database (via the web) assumes their creation had failed. If this occurs, these untracked containers can pile up on the worker and use resources. #3600 ensures that they get cleaned appropriately. #3600
```

Comment:
took my best guess at what was intended, but it looks like there's some weird copy-paste going on here.
```suggestion
 * Concourse now garbage-collects worker containers and volumes that are not tracked in the database. In some niche cases, it is possible for containers and/or volumes to be created on the worker, but the database (via the web) assumes their creation had failed. If this occurs, these untracked containers can pile up on the worker and use resources. #3600 ensures that they get cleaned appropriately.